topo/topo.go

```diff
@@ -116,10 +117,6 @@ func WithBasePath(s string) Option {
 
 // New creates a new topology manager based on the provided kubecfg and topology.
 func New(kubecfg string, pb *tpb.Topology, opts ...Option) (TopologyManager, error) {
```

Comment:
what is the pb argument used in this func for?

instead we are specifying potentially a different pb  with options?

Reply:
Yes, the pb specified will be replaced if WithTopology() is in the Option.

This New() with non-nil pb is still used by KNE cli commands. I will have another PR to revise those code and remove pb from the function signature to avoid confusion..

Reply:
Cool, can you file a cleanup task in the backlog for this additional work?